Dockerfile

```diff
@@ -239,4 +239,18 @@ LABEL maintainer="support@semgrep.com"
 # on the mounted volume when using instructions for running semgrep with docker:
 # `docker run -v "${PWD}:/src" -i returntocorp/semgrep semgrep`
 FROM semgrep-cli AS nonroot
+
+# Remove previous `osemgrep` symlink
```

Comment:
you can remove those comments, they just paraphrasing what is below and so add extra things to read without any benefit.

A comment should be useful, such as explaining a design decision, explaining why we do it this way.
In this case maybe something like
"Without the adjustment belows, a user would not be able to run semgrep --install-pro in a nonroot docker image
because semgrep-core would be in /usr/local/bin which is protected. This is why we need to move it to a non-protected place.
alt: we could also do this work directly it in the root docker image
"